src/quo2/theme.cljs

```diff
@@ -13,4 +13,12 @@
 
 (defn set-theme
   [value]
-  (reset! theme value))
\ No newline at end of file
+  (reset! theme value))
+
+(defn theme-value
+  "Returns a value based on the current/override-theme theme."
+  ([light-value dark-value]
+   (theme-value light-value dark-value nil))
+  ([light-value dark-value override-theme]
+   (let [theme (or override-theme (get-theme))]
+     (if (= theme :light) light-value dark-value))))
```

Comment:
Tiny one: just missing the newline character on line 24.

Reply:
Done. Missed configuring that in my new editor.